demos/simple/output.html

```diff
@@ -7,12 +7,13 @@
 <body>
     <div id="live-editor-output"></div>
 
-    <script src="../../build/js/live-editor.core_deps.js"></script>
+    <!-- TODO(kevinb7) use JSON dependency loading system for core_deps.js and shared.js -->
+    <script src="../../build/js/live-editor.core_deps.js?cachebust=1"></script>
     <script src="../../build/js/live-editor.shared.js"></script>
     <script src="../../build/js/live-editor.output_pjs_deps.js"></script>
     <script src="../../build/js/live-editor.output.js"></script>
     <script src="../../build/js/live-editor.output_pjs.js"></script>
-    <script src="../../build/js/live-editor.output_debugger_deps.js"></script>
+    <!--<script src="../../build/js/live-editor.output_debugger_deps.js"></script>-->
```

Comment:
need to optimize the dependencies so that we're sharing esprima.js which is loaded as part of deps.json.